src/components/exercises/details.cjsx

```diff
@@ -146,7 +146,7 @@ ExerciseDetails = React.createClass
         />
       </div>
       {<div className="page-navigation next" onClick={@onNext}>
-        <div className='triangle' />
+        <Icon type='angle-right' />
        </div> if moves.next}
```

Comment:
I think this is the cause of the missing gutters.  Pretty sure the `.triangle` class set them. Take a look at `mixins/page-navigation-arrows.less`